test/test_game.py

```diff
@@ -380,6 +380,38 @@ def test_next_turn(self, initial_player, initial_remaining_moves,
         self.assertEqual(actual_player.name, expected_player)
         self.assertEqual(actual_remaining_moves, expected_remainig_moves)
 
+    def test_is_game_over_for_incorrect_moves_p1(self):
+        game = WumpusGame()
+        game.player_1.invalid_moves_count = 5
+        message = 'GAME OVER - Player 1 reached 5 invalid moves in a row'
+        self.assertEqual(game.is_game_over(), message)
+
+    def test_is_game_over_for_incorrect_moves_p2(self):
+        game = WumpusGame()
+        game.player_2.invalid_moves_count = 5
+        message = 'GAME OVER - Player 2 reached 5 invalid moves in a row'
+        self.assertEqual(game.is_game_over(), message)
+
+    def test_is_game_over_for_no_turns_left(self):
+        game = WumpusGame()
+        game.remaining_moves = 0
+        message = "GAME OVER - No turns left"
+        self.assertEqual(game.is_game_over(), message)
```

Comment:
can you merge these tests into one?

Reply:
Suggested changes have been implemented. Thank you.